packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -46,6 +46,9 @@ const getStyles = makeStyles({
 		paddingBottom: "2px",
 		verticalAlign: "middle",
 	},
+	inlineValue: {
+		whiteSpace: "pre-line",
```

Comment:
These options are new to me, but based on docs I think we want `pre-wrap` instead? If the user has sequences of whitespace in their string, or leading/trailing whitespace, I think we want to preserve that in the display.

Reply:
I found this example in the docs useful to understand the differences: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space#examples